server/src/controllers/courseInstance.ts

```diff
@@ -125,21 +149,16 @@ export async function getAllCourseInstances(req: Request, res: Response): Promis
 
   const course: Course = await findCourseById(courseId, HttpCode.NotFound);
 
-  // TODO: ADD FUNCTIONALITY FOR MULTIPLE RESPONSIBLE TEACHERS THROUGH COURSE ROLE
   const instances: Array<CourseInstanceWithTeachers> = await CourseInstance.findAll({
-    attributes: [
-      'id', 'sisuCourseInstanceId', 'courseId', 'gradingType', 'startingPeriod',
-      'endingPeriod', 'teachingMethod', 'responsibleTeacher', 'minCredits',
-      'maxCredits', 'startDate', 'endDate', 'createdAt', 'updatedAt'
-    ],
     where: {
       courseId: courseId
     },
-    // TODO: CHANGE TO GO THROUGH COURSE ROLE INSTEAD OF GOING THROUGH
-    // RESPONSIBLE TEACHER FOREIGN KEY
     include: {
       model: User,
-      as: 'teacher'
+      attributes: ['name'],
```

Comment:
This would have many of the fields in the `User` being `null/undefined`, which would violate the contract of the `User` type.